apps/institution-service/src/database.ts

```diff
@@ -10,13 +10,19 @@ const config: { [key: string]: string } = require(
   // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
 )[env];
 
-export default new Sequelize(
-  config.database,
-  config.username,
-  config.password,
-  {
-    dialect: config.dialect as Dialect | undefined,
-    host: config.host,
-    port: config.port as unknown as number,
-  },
-);
+const sequelize = process.env.DATABASE_URL
+  ? new Sequelize(process.env.DATABASE_URL, {
+      dialectOptions: {
+        ssl: {
+          required: true,
+          rejectUnauthorized: false,
```

Comment:
why is rejectUnauthorized false?

Reply:
When it's not false postgres wont allow the connection, I get this error in the heroku logs: 
`Error syncing database: ConnectionError [SequelizeConnectionError]: unable to get local issuer certificate`

I've tried enabling automatic certificate management on heroku and been looking for a way to run it with `rejectUnauthorized: true` but I've had no success yet. I'll keep looking.

It sounds like using `rejectUnauthorized: true` helps prevent man in the middle attacks, but since we don't have any sensitive data it might not matter. But if it's possible to get it to work then we might as well do it.

Reply:
From my research it seems like I can't set rejectUnauthorized to true because the `heroku postgres database` is a private database service which uses self-signed certificates which don't meet the standards for the public Certificate Authority. This is intentional because it gives heroku more flexibility to manage and update certificates internally without relying on the external Certificate Authority. So it sounds to me like we just kind of have to trust that heroku knows what it's doing with ssl. 

So we should still be getting encryption and security from heroku. From this research I feel like it's ok leave this setting as false. What do you think?